pkg/parquet/reader.go

```diff
@@ -30,6 +30,8 @@ type ReadResult map[string]interface{}
 type ReadResults []ReadResult
 type ResultCallback func(progress Progress, results interface{}) (bool, error)
 
+var S3FilesToDelete []string
```

Comment:
I don't like this, but essentially need a way of the writer being able to access the files to delete. Alternatively, I could add the file name onto each spend event item, and then keep track of the files to delete that way... Then I could just pass in a slice to the DeleteFilesInS3...

Reply:
will discuss with Flo in chat